Packs/Core/ReleaseNotes/3_0_96.md

```diff
@@ -0,0 +1,11 @@
+#### Integrations
+
+##### Investigation & Response
+
+- Updated the Docker image to: *demisto/google-cloud-storage:1.0.0.117186*.
+- Added two new optional arguments to the ***core-run-script-execute-commands*** command:
+
+| **Argument Name** | **Type** | **Description** |
+| --- | --- | --- |
+| is_raw_command | Boolean | Determines if shell commands are passed in their raw form or split by commas. Default is "false". |
+| command_type | Predefined | Type of shell command. Possible values: "powershell", "null". Default is "null". |
```

Comment:
Is this necessary?  We usually just list the name of the arguments.  Like this:
- Added two new optional arguments to the ***core-run-script-execute-commands*** command:
   - *is_raw_command*
   - *command_type*